### DISCUSSION TITLE
Deploy to Fly.io

### DISCUSSION THREAD

**Discussion by kennetpostigo** [withastro/roadmap#194] [Proposal]
Hi,

I wanted to know if there are any plans to add [fly.io](https://fly.io/) as a deployment target for Astro applications?

**Comment by delucis** [comment]
I assume this would be for SSR sites, right? Looking at [the fly.io docs](https://fly.io/docs/getting-started/node/), it looks like you can deploy a fairly standard Node.js server with Express and that’s also how you can use [Astro’s Node “adapter”](https://github.com/withastro/astro/tree/main/packages/integrations/node), so I guess this is already supported? What extra steps would be needed?

**Reply by kennetpostigo** [reply]
Hi @delucis ,

Yep, this would be for SSR, i'm currently using Deno on Fly, and I was under the impression that in order to deploy to providers there would need to be a similar adapter like Netlify/Vercel/etc.

If this isn't the case(it seems like it's not the case) then sorry for the noise!

**Reply by delucis** [reply]
No worries! There is a Deno adapter too, so maybe that would work? https://github.com/withastro/astro/tree/main/packages/integrations/deno
